### DISCUSSION TITLE
How can I find my own repos?

### DISCUSSION THREAD

**Discussion by bergamin** [community/community#124182] [Repositories] [answered]
### Select Topic Area

Question

### Body

It's been some time since I last used GitHub. I've been exclusively using GitLab the last few years for my more professional grade work, but now I want to use a tool that sadly only supports GitHub. I've got to the home page and I can't find anything anymore... To the left the repos I have contributed before. To the right, random repositories and in the middle some updates from starred repos. The side menu doesn't help either. When I go to "Projects", it doesn't list anything...

What happened to this website since the last time I've been here?

**Comment by CopperEagle** [comment, marked as answer]
Hey there @bergamin, welcome back!

One way to do it is to click on your profile image in the top right corner. This opens the right sidebar. 
![Sidebar](https://github.com/community/community/assets/137458753/53ab7fb1-5196-4f7b-a281-b0f9f33ae3c0)


Then, go to `Your Profile` which is the second option from the top. There you have several tabs at the top. The one you need to click is `Repositories`, which is the second tab right next to `Overview`.
![Profile](https://github.com/community/community/assets/137458753/08c2878f-87ea-4c54-b552-f9acccd21e25)


![Repositories](https://github.com/community/community/assets/137458753/0f801ad5-3fe9-475c-ba29-9c8f8750eb0d)

Note that repositories are not projects themselves. That's two different things.
Hope this helps.

**Reply by bergamin** [reply]
Thanks, @CopperEagle! That helped a lot!
